Packs/Base/ReleaseNotes/1_18_2.md

```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
 ##### CommonServerPython
-- Maintenance and stability enhancements.
\ No newline at end of file
+- Maintenance and stability enhancements.
```

Comment:
Why is there a change here?

Reply:
I added an extra line by mistake.
Fixed: https://github.com/demisto/content/pull/16776/commits/08fbf879aef16aa4d7b273d0d50c438661e984db